UNRELEASED.md

```diff
@@ -2,6 +2,15 @@
 
 > Place your changes below this line.
 
+**Added:**
+- bpk-svgs:
+  - New `center-location` icon.
```

Comment:
I think this probably got added in a rebase?

Reply:
🤦

Reply:
Removed 👍